builder/image/Dockerfile

```diff
@@ -61,5 +52,7 @@ ADD etc /etc
 ADD . /app
 RUN chown -R root:root /app
 
+RUN /app/bin/install-buildpacks.sh
```

Comment:
@johanneswuerbach I just did that to keep control over the buildpack versions. I don't have any opinions about this.